### PR TITLE
Next dev

### PR DESCRIPTION
**Pull request by JonasCir**
@jabrandes 
@MichaelSp 

This is the layout for the next dev branch. The server code is removed, the PoC for SORMAS is included. 

- This approach decouples this repo from the SORMAS fork. The idea is to run SORMAS via docker-compose and our client seperately. This makes it easier to upstream changes to SORMAS repos.
- ~I decided to keep the stuff in client as we might add some more top level folders for docs etc. in the future. This way we keep the structure clean~ -> Moved to root
-  ~I fixed the gradle build files. They work for the current layout, but likely need some attention in the future, especially in the context of CI~ -> removed
- ~Working CI is not working yet~
- There is no Client generation in the build process. For now, SormasSwaggerApi.ts is shipped.
- The old ImisSwagger file is kept to ensure passing builds

### CODE REVIEW

File: client/build.gradle [hunk 1]
```diff
@@ -21,40 +21,40 @@ def sourceSet = fileTree(dir: '.', includes: [
...
 
 task serve(type: YarnTask) {
-  setDescription "Run a local frontend development server."
-  args = ['serve']
+	setDescription "Run a local frontend development server."
+	args = ['serve']
 }
 
-task generateClient(type: NpxTask) {
+/*task generateClient(type: NpxTask) {
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply client/build.gradle:46]: @jabrandes I hope you PR in SORMAS gets merged soon, then we can think about ways how to enable it again

**Comment by JonasCir** [comment]
@jabrandes good point, I think I created a tag. but I'm not familiar how GitHub handles them.
Please feel free to create a tag or branch head, I'm not familiar with the latter one.

**Comment by jabrandes** [comment]
@JonasCir, I am not that familiar with tags, but it I know that it's quite easy to create a new branch. You could use
```
git branch <brachname>
```
to create a new branch and stay on the current, or use
```
git checkout -b <branchname>
```
to create a new branch and check it out at once. The current HEAD commit will be head of the newly created branch.

**Comment by MichaelSp** [comment]
I'd opt for removing `server` and gradle and moving `client` to root-dir. Why?

1. If anyone is interested in the history: We are using git. It's there.
2. New developers don't know about our past and expect a client frontend only repo, where the package.json is in the root. So I'd even drop gradle.
3. Most CI-tasks are easier if you are not using a custom folder layout.

**Comment by JonasCir** [comment]
@jabrandes Ok I didn't understand, that you meant a new branch :smile: 
@MichaelSp Good points, gradle is gone and client is now in the root folder

File: Dockerfile.prod [hunk 2]
```diff
@@ -8,7 +8,7 @@ RUN yarn run build
 
 # production stage
 FROM nginx:stable-alpine as production-stage
-COPY --from=build-stage /app/dist /usr/share/nginx/html
-EXPOSE 80
+RUN sed  "s/80;/8080;/g" /etc/nginx/conf.d/*.confCOPY --from=build-stage /app/dist /usr/share/nginx/html
```
Comment (JonasCir) [inline Dockerfile.prod:11]: Missing newline betwwen .conf and COPY ? @MichaelSp :)

Comment (MichaelSp) [reply Dockerfile.prod:11]: oh yeah...😊

**Comment by MichaelSp** [comment]
tag created: https://github.com/ImisDevelopers/1_011_a_infektionsfall_uebermittellung/tree/before-move-to-sormas

**Pull request merged**